Statement/Statement.php

```diff
@@ -319,62 +319,32 @@ public function removeTransaction(TransactionInterface $transaction)
         return $this;
     }
 
-    /**
-     * @see \Countable::count()
-     *
-     * @return int
-     */
-    public function count()
+    public function count(): int
     {
         return count($this->transactions);
     }
 
-    /**
-     * @see \Iterator::current()
-     *
-     * @return TransactionInterface
-     */
-    public function current()
+    public function current(): mixed
```

Comment:
Refactor this class to implement IteratorAggregate instead of Iterator.

Reply:
replaced